pwndbg/commands/telescope.py

```diff
@@ -39,7 +39,7 @@ def telescope(address=None, count=telescope_lines, to_string=False):
     ($sp by default)
     """
     address = int(address if address else pwndbg.regs.sp) & pwndbg.arch.ptrmask
-    count   = int(count) & pwndbg.arch.ptrmask
+    count   = max(int(count), 0) & pwndbg.arch.ptrmask
```

Comment:
This isn't actually what we want, and breaks the behavior of `telescope A B` to dump everything from `[A..B]`